ocaml/testsuite/tests/typing-layouts-err-msg/immediate.ml

```diff
@@ -0,0 +1,66 @@
+(* TEST
+   flags = "-extension layouts_alpha"
+   * expect
+*)
+
+(***************************)
+(* Immediate layout errors *)
+
+(* Empty_record *)
+(* Records with all void fields are not yet supported *)
```

Comment:
Make the test anyway. The error will be about void elements in a record, but then it will magically become the right error later.